arch/lkl/scripts/headers_install.py

```diff
@@ -138,6 +138,11 @@ def replace(h):
 
 new_headers = set()
 
+# Create symlink of gsed as sed on FreeBSD
+if 'freebsd' in sys.platform:
+    print('Creating symlink of gsed as sed')
+    os.system('ln -sf `which gsed` '+srctree+'/tools/lkl/bin/sed')
+
```

Comment:
do you think it's possible to put those into `tools/lkl/Makefile`, as `bin/stat` target ?  (https://github.com/lkl/linux/blob/master/tools/lkl/Makefile#L55)

Reply:
You are right. It is cleaner in Makefile. 

It is just weird on my system. Sometimes gmake fails to generate the symlink and only leaves a bin/stat without actually linking to `which gnustat`. That was the reason why I reported gnustat in issue #443 . The issue usually disappears after I start to build in a new and clean directory of source codes.

Anyway, I will use the Makefile method and change my patch.

Reply:
okay.

I'm not sure but if the make target `bin/stat` fails and accidentally create an invalid `bin/stat` file, then make command won't execute this target no ?

Reply:
Yeah it throws an error of stat "invalid argument" (of course because bin/stat is just a dummy file without symlink) and stops the build. I am not able to repeat this bug, and most of the time build it again in a new directory of source codes will work just fine.

Reply:
ack.

and, please squash two commits into a single one (and force push to your repository) so that resulting commits log will be simplified.

Reply:
Do you mean squashing all 3 commits or just the first two?

Reply:
I meant two commits: readme part and the other are fine.

Reply:
Done.